README.md

```diff
@@ -1 +1,31 @@
 golang-librdkafka
+
+## Alpine
+
+Build and tag (from within the directory that contains the Dockerfile):
```

Comment:
Since you are already setting all these versions, you can probably build the directory path as well, right? Then you can just run the command from the root of the repository.

Reply:
Good plan.